test/semicircle.jl

```diff
@@ -37,3 +38,36 @@ d = Semicircle(2.0)
 @test quantile(d,  .0) == -2.0
 @test quantile(d,  .5) ==   .0
 @test quantile(d, 1.0) == +2.0
+
+function test_sample_uniform(rng,sample)
```

Comment:
This is a bit ad hoc... I will check the literature if there are statistical tests for uniform distribution that are easy to implement and powerful.

Reply:
Is it an option to have `HypothesisTests.jl` as a test dependency?

Reply:
No, I would like to avoid that. We removed it a while ago because it was very annoying since it created a circular dependency. So, e.g., it was not possible to test any breaking releases properly.

Reply:
However, you could use `pvalue_kolmogorovsmirnoff` which was added to the tests when the HypothesisTests tests dependency was removed: https://github.com/JuliaStats/Distributions.jl/blob/8864b314c29925d6cf512965ace6818e600675b5/test/testutils.jl#L594-L597